packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -93,6 +93,7 @@ function defaultRequestToExternalModule( request ) {
 	switch ( request ) {
 		case '@wordpress/interactivity-router':
 		case '@wordpress/a11y':
+		case '@wordpress/polyfill':
```

Comment:
This should probably be in the condition above. `import …` would make it async. `module …` makes it a static import. With a polyfill, it would needs to be evaluated.

But if we don't have a module for this… I'm not sure what to do 😕

Reply:
One note. As far as I’m aware Interactivity API doesn’t use any polyfills for runtime and router on production. I doubt it ever had it integrated.

Reply:
Changing this code shouldn't be necessary anymore. Thanks for the help!